webdriver/state/text/get_text.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import uuid
+
+from support.asserts import assert_error, assert_success
+from support.inline import inline
+
+def test_getting_text_of_a_non_existant_element_is_an_error(session):
+   session.url = inline("""<body>Hello world</body>""")
+   id = uuid.uuid4()
+
+   result = session.transport.send(
```

Comment:
OK, I guess this is clearer than using `pytest.raises` and assuming that the error text is correct in the client.